about-us.html

```diff
@@ -0,0 +1,137 @@
+                <path fill-rule="evenodd" d="M4 2a2 2 0 00-2 2v11a3 3 0 106 0V4a2 2 0 00-2-2H4zm1 14a1 1 0 100-2 1 1 0 000 2zm5-1.757l4.9-4.9a2 2 0 000-2.828L13.485 5.1a2 2 0 00-2.828 0L10 5.757v8.486zM16 18H9.071l6-6H16a2 2 0 012 2v2a2 2 0 01-2 2z" clip-rule="evenodd" />
+              </svg>
+            </a>
+          </div>
+          <div class="main-menu">
+            <ul>
+              <li><a href="./index.html"> Home </a></li>
+              <li><a href="./about-us.html" class="last-child"> About Us </a></li>
+            </ul>
+          </div>
+          <div class="menu-icon">
+            <svg height="20" width="20" xmlns="http://www.w3.org/2000/svg" class="h-6 w-6" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+              <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M4 6h16M4 12h16M4 18h16" />
+            </svg>
+          </div>
+        </nav>
+      </div>
+      <div class="mobile-navigaton hide">
+        <div class="close-btn">
+          <svg height="20" width="20" xmlns="http://www.w3.org/2000/svg" class="h-6 w-6" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+            <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M6 18L18 6M6 6l12 12" />
+          </svg>
+        </div>
+        <div class="mobile-menu">
+          <ul>
+            <li> <a href="./index.html">Home</a></li>
+            <li> <a href="./about-us.html">About Us </a></li>
+          </ul>
+        </div>
+      </div>
+    </section>
+    <section id="heading">
+      <div class="head">
+        <div class="container">
+          <div class="head-content">
+            <h2 class="title-about">Tourism Events Global summit <span class="year"> 2022 </span></h2>
+
+            <div class="head-text-about">
+              <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Blanditiis, illum nostrum maiores qui distinctio quam asperiores facere dolore sint amet ex optio, doloribus exercitationem magni, enim tempora itaque ratione tempore?
+                Architecto ex perferendis aspernatur consequatur iste inventore sunt sequi. Voluptatibus dolores quas quibusdam? Qui perspiciatis aliquid nemo, excepturi in, nulla quas ad ducimus iure facere inventore aliquam iste repudiandae tempore?
+                tempora itaque ratione tempore? Architecto ex perferendis aspernatur consequatur iste inventore sunt sequi. Voluptatibus dolores quas quibusdam? Qui perspiciatis aliquid nemo, excepturi in, nulla quas ad ducimus iure facere inventore aliquam iste repudiandae tempore?
+                tempora itaque ratione tempore? Architecto ex perferendis aspernatur consequatur iste inventore sunt sequi. Voluptatibus dolores quas quibusdam? Qui perspiciatis aliquid nemo, excepturi in, nulla quas ad ducimus iure facere inventore aliquam iste repudiandae tempore?
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
```

Comment:
- [ ] There is an entire middle section that you are missing here it comes right after this section. it's in the middle of the page before you get to the section with the two images. please add this section to your about page. 
## Reference image (👇 you should have this section of the page)
![image](https://user-images.githubusercontent.com/18730106/147155557-4b101c15-5e1f-4955-8eb3-c38d5fbe37bd.png)